db/migrate/20160223195539_migrate_network_partner_participants_to_facilitators.rb

```diff
@@ -0,0 +1,34 @@
+class MigrateNetworkPartnerParticipantsToFacilitators < ActiveRecord::Migration
```

Comment:
this is a data migration, it depends whether the **other branch** depends on the sate this migration is modifying(network partner roles in assessments).